src/main/java/pl/javastart/task/Main.java

```diff
@@ -3,9 +3,10 @@
 import java.util.Scanner;
 
 public class Main {
+    static final int CELL_SIZE = 7;
 
     public static void main(String[] args) {
-        final int cellSize = 7;
```

Comment:
@marcinkunert  Nazwałem stałą tak jak poleciłeś, ale zaczął się wywalać checkstyle. To jest błąd w checkstyle czy w nazwie zmiennej? 
![checkstyle](https://github.com/javastartbootcamp/java-alior_afir123_zad_4.6/assets/74791259/225f7bc9-1f2a-4378-b47f-7ee768c8a977)